extensions/positron-python/python_files/positron/positron_ipykernel/ui.py

```diff
@@ -186,26 +186,18 @@ def open(self, url, new=0, autoraise=True):
         if not self._comm:
             return False
 
+        is_plot = False
         # If url is pointing to an HTML file, route to the ShowHtmlFile comm
         if is_local_html_file(url):
-            is_plot = False
 
             # Send bokeh plots to the plots pane.
             # Identify bokeh plots by checking the stack for the bokeh.io.showing.show function.
             # This is not great but currently the only information we have.
-            bokeh_io_showing = sys.modules.get("bokeh.io.showing")
-            if bokeh_io_showing:
-                for frame_info in inspect.stack():
-                    if (
-                        inspect.getmodule(frame_info.frame, frame_info.filename) == bokeh_io_showing
-                        and frame_info.function == "show"
-                    ):
-                        is_plot = True
-                        break
-                if os.name == "nt":
-                    url = urlparse(url).netloc or urlparse(url).path
-
-            self._comm.send_event(
+            is_plot = self._is_module_function("bokeh.io.showing", "show")
+            if os.name == "nt":
+                url = urlparse(url).netloc or urlparse(url).path
+
+            return self._comm.send_event(
```

Comment:
I think `webbrowser` expects this method to return a boolean: `True` if we've handled the request.

Reply:
Should we be calling `_send_show_html_event` here?